labrad/gpib.py

```diff
@@ -88,6 +88,19 @@ def write(self, s, timeout=None):
         returnValue(resp.write)
 
     @inlineCallbacks
+    def write_raw(self, s, timeout=None):
+        """Write a string to the device."""
+        p = self._packet()
+        if timeout is not None:
+            p.timeout(timeout)
+        p.write_raw(s)
```

Comment:
The [gpib server](https://github.com/martinisgroup/servers/blob/master/gpib_server.py) does not yet have a write_raw setting, so this will not work until the server is updated as well. I do think it would be a good idea to add that.